packages/framework/fluid-static/src/types.ts

```diff
@@ -228,16 +228,16 @@ export interface IMember {
 	/**
 	 * An ID for the user, unique among each individual user connecting to the session.
 	 */
-	userId: string;
+	readonly userId: string;
 
 	/**
 	 * The set of connections the user has made, e.g. from multiple tabs or devices.
 	 */
-	connections: IConnection[];
+	readonly connections: IConnection[];
```

Comment:
Making this array read-only introduced compile errors in our implementation since it was storing data in the `IMember` format so I have left this array mutable.

Reply:
We should probably leave a TODO

Reply:
I have a list of API issues (which these read-only fixes were part of) in https://github.com/microsoft/FluidFramework/pull/19724 which will continue to track this, and I believe that is tracked by a work item, so thats not getting lost, simply not fixed in this change.